### PR TITLE
Beers Structure

### PR DESCRIPTION
**Pull request by diegosaul402**
This PR adds the basic file structure for the beer model. Creates the model, migration, routes and a very basic view for beer/index.
Also adds some configuration for factory bot and pry with rspec.

### CODE REVIEW

File: app/assets/javascripts/beer.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (jclopezdev) [inline app/assets/javascripts/beer.js]: Please delete these files if you are not going to use them

File: app/assets/javascripts/beers.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (jclopezdev) [inline app/assets/javascripts/beers.js]: Please delete these files if you are not going to use them

File: app/assets/stylesheets/beers.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Beers controller here.
```
Comment (jclopezdev) [inline app/assets/stylesheets/beers.scss]: Please delete these files if you are not going to use them

**Review by jclopezdev — CHANGES_REQUESTED** [review]

**Comment by jnegrete** [comment]
you have potential security vulnerabilities in two of your dependencies, you should do the following changes in your Gemfile.lock:

loofah (2.2.0)          to          loofah (2.2.1)
and
rails-html-sanitizer (1.0.3)          to          rails-html-sanitizer ~> 1.0.4

see:
flavorjones/loofah#144
and
https://nvd.nist.gov/vuln/detail/CVE-2018-3741

**Review by jnegrete — APPROVED** [review]
LGTM

**Review by jclopezdev — APPROVED** [review]
LGTM!

**Pull request merged**